### PR TITLE
🐛 Update to rrule 2.6.2 to Fix DoS Vulnerabilities

### PR DESCRIPTION
**Pull request by PatOnTheBack**
This fixes a [high severity DoS vulnerability found by Snyk](https://app.snyk.io/vuln/SNYK-JS-RRULE-72455).
This also fixes a [medium severity DoS vulnerability found by Snyk](https://app.snyk.io/vuln/SNYK-JS-RRULE-72421).

### CODE REVIEW

**Comment by PatOnTheBack** [comment]
I signed it!

File: package.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
     "promise-pjs": "1.1.3",
     "prop-types": "15.7.2",
     "react-dates": "15.5.3",
-    "rrule": "2.2.0",
+    "rrule": "2.6.2",
```
Comment (rsimha) [inline package.json]: This needs an accompanying change in `yarn.lock`. You can generate it by running `yarn`.

Comment (PatOnTheBack) [reply package.json]: @rsimha I tried running `yarn`, but it did not work properly in my local environment.

Am I supposed to set up my own Travis CI environment or run the command locally on my computer?

Is there a way someone else can run yarn to modify my PR?

Comment (rsimha) [reply package.json]: Assigned this to @cvializ

**Comment by cvializ** [comment]
Did the git-fu to add yarn.lock

**Comment by PatOnTheBack** [comment]
I never tried `git-fu`. I trued `gulp update-packages` and also just `yarn`. Neither worked.

**Comment by cvializ** [comment]
Ah sorry I was unclear, I meant I did the git commands to check out your branch, run `yarn` and commit and push the `yarn.lock` change. You'd have to share what the output was when it didn't work for me to help you, but the lockfile is committed now

**Comment by cvializ** [comment]
@rsimha PTAL

**Pull request merged**